adapters/balance.js

```diff
@@ -52,13 +60,8 @@ module.exports = function (mu, transports) {
     muid: muid,
     tf: tf,
     type: 'transport',
-    setId: setId
-    // driver: driver,
-    // tearDown: tearDown
+    setId: setId,
```

Comment:
@pelger can you please elaborate on the overall signature of this object?I am a little bit confused by the object contract by the using of explicit *setter* (or look alike!) functions.